### PR TITLE
Add new functions DeleteSlicer and GetSlicers

### PR DESCRIPTION
**Pull request by zhangyimingdatiancai**
# PR Details

1. Create a new `GetSlicers` function to get the slicer definition list
2. Create a new `DeleteSlicer` function to delete specified slicer by the given slicer name
3. Implement this feature in the `slicer.go` and `xmlSlicers.go` source code

## Description

Slicers in spreadsheets are filter tools used along with Excel tables or pivot tables over a large amount of data. Currently, the Excelize library already supports to creation of a slicer for the table or pivot table in the worksheet. The goal of this PR is to add the function for getting and deleting slicers and support for removing slicers by the given slicer name.

## Related Issue

N/A

## Motivation and Context

Add get and delete slicer support

## How Has This Been Tested

Complete documentation and unit tests, written unit test code in the `slicer_test.go` source code, incremental test coverage of line reach 100%

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: xmlDrawing.go [hunk 1]
```diff
@@ -325,6 +325,28 @@ type xlsxChart struct {
 	R   string `xml:"xmlns:r,attr"`
 }
 
+// decodeChoice defines the structure used to deserialize the mc:Choice element.
```
Comment (xuri) [inline xmlDrawing.go:328]: This code block should be move into `xmlDecodeDrawing.go`.

Comment (zhangyimingdatiancai) [reply xmlDrawing.go]: I moved this code block, you can take a look again.

File: slicer.go [hunk 2]
```diff
@@ -725,3 +739,201 @@ func (f *File) addWorkbookSlicerCache(slicerCacheID int, URI string) error {
 	wb.ExtLst = &xlsxExtLst{Ext: strings.TrimSuffix(strings.TrimPrefix(string(extLstBytes), "<extLst>"), "</extLst>")}
 	return err
 }
+
+// GetSlicers provides the method to get all slicers in a worksheet by a given
```
Comment (xuri) [inline slicer.go:743]: Good job. I think the `GetSlicers` function is almost done, please add unit tests for this.

**Review by xuri — CHANGES_REQUESTED** [review]

File: slicer.go (hunk 2 above)
Comment (zhangyimingdatiancai) [reply slicer.go:743]: Thanks for the review. The unit tests for the `GetSlicers` function has been added.

**Comment by zhangyimingdatiancai** [comment]
Hi @xuri, the `DeleteSlicer` function now has been implemented. I added unit tests for it. This PR ready for review.

File: pivotTable.go [hunk 3]
```diff
@@ -785,15 +785,14 @@ func (f *File) getPivotTableDataRange(opts *PivotTableOptions) error {
...
-			return err
-		}
-		for _, table := range tables {
-			if table.Name == opts.DataRange {
-				opts.pivotDataRange, opts.namedDataRange = fmt.Sprintf("%s!%s", sheetName, table.Range), true
+	tbls, err := f.getTables()
+	if err != nil {
+		return err
+	}
+	for sheetName, tables := range tbls {
```
Comment (xuri) [inline pivotTable.go:792]: There're some blank prefix characters in these lines, please format source code by `gofmt`.

Comment (zhangyimingdatiancai) [reply pivotTable.go:792]: Done.

**Review by xuri — APPROVED** [review]
LGTM, thanks for your contribution.

**Pull request merged**